implementation/endpoints/include/netlink_connector.hpp

```diff
@@ -20,6 +20,28 @@
 
 #include "../../endpoints/include/buffer.hpp"
 
+inline bool operator<(const sockaddr_nl &rhs, const sockaddr_nl &lhs)
+{
+    if (rhs.nl_family < lhs.nl_family &&
+            rhs.nl_pad < lhs.nl_pad &&
+            rhs.nl_pid < lhs.nl_pid &&
+            rhs.nl_groups < lhs.nl_groups)
```

Comment:
By the way, the equality operator is easy enough, but is this implementation of < really correct?
What about, for example,  rhs.nl_family < lhs.nl_family but one of pad, pid, groups are equal?  Is that not also less than?  Did you find an example online how to write this operator correctly?